datastore/system-test/concepts.test.js

```diff
@@ -1,282 +1,149 @@
     entity = new Entity(projectId);
     query = new Query(projectId);
   });
 
-  after(function (done) {
-    var datastore = transaction.datastore;
-    var query = datastore.createQuery('Task');
-
-    testUtil.deleteEntities(datastore, query, done);
-  });
-
-  // Transactions
-
-  it('performs a transactional update', function (done) {
-    transaction.testTransactionalUpdate(done);
-  });
-
-  it('performs retries if necessary', function (done) {
-    transaction.testTransactionalRetry(done);
-  });
-
-  it('performs a get or create', function (done) {
-    transaction.testTransactionalGetOrCreate(done);
-  });
-
-  it('gets a snapshot of task list entities', function (done) {
-    transaction.testSingleEntityGroupReadOnly(done);
-  });
-
-  // Metadata
-
-  it('performs a namespace query', function (done) {
-    metadata.testNamespaceRunQuery(done);
-  });
-
-  it('performs a kind query', function (done) {
-    metadata.testKindRunQuery(done);
-  });
-
-  it('performs a property query', function (done) {
-    metadata.testPropertyRunQuery(done);
-  });
-
-  it('performs a property by kind query', function (done) {
-    metadata.testPropertyByKindRunQuery(done);
-  });
-
-  // Indexes
-
-  it(
-    'performs a query with a filter on an unindexed property',
-    function (done) {
-      index.testUnindexedPropertyQuery(done);
-    }
-  );
-
-  it('inserts arrays of data', function (done) {
-    index.testExplodingProperties(done);
-  });
-
-  // Queries
-
-  it('performs a basic query', function (done) {
-    query.testRunQuery(done);
-  });
-
-  it('performs a query with a property filter', function (done) {
-    query.testPropertyFilter(done);
-  });
-
-  it('performs a query with a composite filter', function (done) {
-    query.testCompositeFilter(done);
-  });
-
-  it('performs a query with a key filter', function (done) {
-    query.testKeyFilter(done);
-  });
-
-  it('performs a query with ascending sort', function (done) {
-    query.testAscendingSort(done);
-  });
-
-  it('performs a query with descending sort', function (done) {
-    query.testDescendingSort(done);
-  });
-
-  it('performs a query with multi sort', function (done) {
-    query.testMultiSort(done);
-  });
-
-  it('performs a kindless query', function (done) {
-    query.testKindlessQuery(done);
-  });
-
-  it('performs a projection query', function (done) {
-    entity.testProperties(function (err, tasks) {
-      console.log(err, tasks);
-      assert.ifError(err);
-      setTimeout(function () {
-        query.testRunQueryProjection(done);
-      }, 1000);
+  after(() => {
+    const datastore = transaction.datastore;
+    const query = datastore.createQuery(`Task`).select(`__key__`);
+    return testUtil.deleteEntities(datastore, query);
+  });
+
+  describe(`Transactions`, () => {
+    it(`performs a transactional update`, () => transaction.testTransactionalUpdate());
+    it(`performs retries if necessary`, () => transaction.testTransactionalRetry());
```

Comment:
This pattern probably needs to be changed to fit our existing test standards - if not now, then later.